invenio/templates/configurations/invenio.yaml

```diff
@@ -4,13 +4,13 @@ kind: ConfigMap
 metadata:
   name: invenio-config
 data:
-  INVENIO_ACCOUNTS_SESSION_REDIS_URL: 'redis://{{ include "redis.host_name" . }}:6379/1'
+  INVENIO_ACCOUNTS_SESSION_REDIS_URL: 'redis://{{ include "redis.host_name" . }}:{{ .Values.redis.master.containerPorts.redis }}/1'
```

Comment:
What would happen if `redis.invenio.enabled = false` (i.e. using an external redis service). Would these variable have incorrect values?

Reply:
Yes, indeed, you are right. Or no, if these settings are part of `values.yaml`. I added it for Redis, RabbitMQ and PostgreSQL to have it consistent.